tools/extension_point_identifiers_parser/extension_point_identifiers_parser.py

```diff
@@ -28,6 +28,13 @@ def _main(
         extension_point_identifier = (
             plist.get("NSExtension", {}).get("NSExtensionPointIdentifier")
         )
+
+        # Supports ExtensionKit
+        if not extension_point_identifier:
+            extension_point_identifier = (
+                plist.get("EXAppExtensionAttributes", {}).get("EXExtensionPointIdentifier")
+            )
```

Comment:
Something like this might be more clear:

```Starlark

ns_extension = plist.get(...)
app_extension = plist.get(...)

if ns_extension:
  ...
elif app_extension:
  ...

```

Reply:
How about iterating through a list of key pairs? It will allow new extension types in future.